Colm

Abstract: Genetic interactions, including synthetic lethal effects, can now be systematically identified in cancer cell lines using high-throughput genetic perturbation screens. Despite this advance, few genetic interactions have been reproduced across multiple studies and many appear highly context-specific. Here, by developing a new computational approach, we identified 220 robust driver-gene associated genetic interactions that can be reproduced across independent experiments and across non-overlapping cell line panels. Analysis of these interactions demonstrated that: (i) oncogene addiction effects are more robust than oncogene-related synthetic lethal effects; and (ii) robust genetic interactions are enriched among gene pairs whose protein products physically interact. Exploiting the latter observation, we used a protein–protein interaction network to identify robust synthetic lethal effects associated with passenger gene alterations and validated two new synthetic lethal effects. Our results suggest that protein–protein interaction networks can be used to prioritise therapeutic targets that will be more robust to tumour heterogeneity.

摘要：利用高通量基因扰动筛选，现在可以在癌细胞系中系统地识别基因相互作用，包括合成致死效应。尽管有这一进展，但很少有基因相互作用在多个研究中得到重复，而且许多基因相互作用似乎高度依赖于特定环境。在本研究中，通过开发一种新的计算方法，我们确定了220个稳定的驱动基因相关基因相互作用，这些相互作用可以在独立实验和非重叠细胞系组中得到重复。这些相互作用的分析表明：（i）癌基因成瘾效应比癌基因相关的合成致死效应更为稳定；（ii）稳定的基因相互作用在蛋白质产物物理相互作用的基因对中富集。利用后一观察结果，我们利用蛋白质相互作用网络识别与乘客基因变异相关的稳定的合成致死效应，并验证了两个新的合成致死效应。我们的结果表明，蛋白质相互作用网络可以用于优先考虑对肿瘤异质性更具稳定性的治疗靶点。

## Introduction

Large-scale tumour genome sequencing efforts have provided us with a compendium of driver genes that are recurrently altered in human cancers (Vogelstein et al., 2013). In some cases, these genetic alterations have been associated with altered sensitivity to targeted therapies. Examples of targeted therapies already in clinical use include approaches that exploit oncogene addictions, such as the increased sensitivity of BRAF mutant melanomas to BRAF inhibitors (Chapman et al., 2011), and approaches that exploit non-oncogene addiction/synthetic lethality, such as the sensitivity of BRCA1/2 mutant ovarian or breast cancers to PARP inhibitors (Lord and Ashworth, 2017). An ongoing challenge is to associate the presence of other driver gene alterations with sensitivity to existing therapeutic agents (Barretina et al., 2012; Iorio et al., 2016) or to identify candidate therapeutic targets whose inhibition may provide therapeutic benefit to patients with specific mutations. Towards this end, multiple groups have performed large-scale loss-of-function genetic perturbation screens in panels of tumour cell lines to identify vulnerabilities that are associated with the presence or absence of specific driver gene mutations (i.e. genetic interactions) (Behan et al., 2019; Campbell et al., 2016; Marcotte et al., 2016; McDonald et al., 2017; Meyers et al., 2017; Tsherniak et al., 2017). Others have performed screens in ‘isogenic’ cell line pairs that differ only by the presence of a specific oncogenic alteration (Martin et al., 2017; Steckel et al., 2012). Despite these large-scale efforts, very few genetic interactions have been identified in more than one study (recently reviewed Ryan et al., 2018). Even in the case of cancer driver genes subjected to multiple screens, such as KRAS, few genetic interactions have been identified in more than one screen (Downward, 2015). This lack of reproducibility may be due to technical issues, for example false positives and false negatives due to inefficient gene targeting reagents (Kaelin, 2012), and/or real biological issues, such as the context specificity of genetic interactions (Henkel et al., 2019; Ryan et al., 2018). We refer to those genetic interactions that can be reproduced across multiple screens and across distinct cell line contexts as robust genetic interactions. Given that tumours exhibit considerable molecular heterogeneity both within and between patients there is a real need to: (i) identify robust genetic interactions that can be reproduced across heterogeneous cell line panels, reasoning that these reproducible effects will be more likely to be robust in the face of the molecular heterogeneity seen in human cancers; (ii) prioritise these robust genetic interactions for further therapeutic development; and (iii) understand the characteristics of robust genetic interactions in cancer as a means to predict new therapeutic targets. To achieve this, we developed, and describe here, a computational approach that leverages large-scale cell line panel screens to identify those genetic interactions that can be reproducibly identified across multiple independent experiments. We found that for all oncogenes studied, the most significant reproducible dependency identified was an oncogene-addiction rather than a synthetic lethal effect. Excluding oncogene addictions, we found 220 reproducible genetic interactions. In investigating the nature of these robust genetic interactions, we found that they are significantly enriched among gene pairs whose protein products physically interact. This suggests that incorporating prior knowledge of protein–protein interactions may be a useful approach to guide the selection of reproducible ‘hits’ from genetic screens as candidates worth considering as therapeutic targets in cancer. We demonstrate the utility of the approach in identifying robust synthetic lethal interactions from chemogenetic screens and in identifying synthetic lethal interactions associated with ‘passenger’ gene alterations.

## 引言

大规模肿瘤基因组测序工作为我们提供了一份经常在人类肿瘤中发生变化的驱动基因汇编（Vogelstein et al.，2013）。在某些情况下，这些基因变异已经与靶向治疗的敏感性改变相关联。已经在临床应用中的靶向治疗方法包括利用癌基因成瘾，例如BRAF突变黑色素瘤对BRAF抑制剂的敏感性增加（Chapman et al.，2011），以及利用非癌基因成瘾/合成致死，例如BRCA1/2突变卵巢或乳腺癌对PARP抑制剂的敏感性（Lord and Ashworth，2017）。一个持续存在的挑战是将其他驱动基因变异的存在与对现有治疗药物的敏感性相关联（Barretina et al.，2012；Iorio et al.，2016），或者识别出其抑制可能为具有特定突变的患者提供治疗益处的候选治疗靶点。为了实现这一目标，多个研究小组在肿瘤细胞系组中进行了大规模的功能丧失基因扰动筛选，以识别与特定驱动基因突变的存在或缺失相关联的易感性（即基因相互作用）（Behan et al.，2019；Campbell et al.，2016；Marcotte et al.，2016；McDonald et al.，2017；Meyers et al.，2017；Tsherniak et al.，2017）。其他人在只有特定致癌突变存在与否不同的“同构”细胞系对中进行了筛选（Martin et al.，2017；Steckel et al.，2012）。尽管进行了这些大规模的努力，**很少有基因相互作用在多个研究中被识别出来**（最近由Ryan等人回顾，2018年）。即使在经过多次筛选的癌症驱动基因的情况下，例如KRAS，在多个筛选中也只识别出了很少的基因相互作用（Downward，2015年）。这种缺乏可重复性可能是由于技术问题，例如由于效率低下的基因靶向试剂导致的假阳性和假阴性（Kaelin，2012年），和/或者是真正的生物问题，例如基因相互作用的上下文特异性（Henkel et al.，2019年；Ryan et al.，2018年）。我们将那些可以在多个筛选和不同细胞系环境中得到重复的基因相互作用称为稳定的基因相互作用。鉴于肿瘤内和患者之间存在显著的分子异质性，我们确实需要：（i）识别可以在异质细胞系组中得到重复的稳定的基因相互作用，认为这些可重复效应在人类肿瘤中看到的分子异质性面前更有可能是稳定的；（ii）为进一步的治疗开发优先考虑这些稳定的基因相互作用；（iii）了解肿瘤中稳定的基因相互作用的特征，以预测新的治疗靶点。为了实现这一目标，我们开发了一种新的计算方法，并在此处描述了它，利用大规模细胞系组筛选来识别那些可以在多个独立实验中重复识别的基因相互作用。我们发现，对于所有研究中研究的癌基因，最显著的可重复依赖性识别是癌基因成瘾，而不是合成致死效应。除去癌基因成瘾，我们发现了220个可重复的基因相互作用。在研究这些稳定的基因相互作用的性质时，我们发现它们在其蛋白产物之间物理相互作用的基因对中明显富集。这表明，蛋白质相互作用网络的引入可能是一种用于指导从遗传筛选中选择可重复“命中”的治疗靶点的有用方法，这些靶点在肿瘤异质性面前更加稳定。我们在化学遗传筛选中识别了稳定的合成致死相互作用，并鉴定了与“乘客”基因改变相关的两个新的合成致死效应，从而验证了这种方法的效用。我们的结果表明，蛋白质相互作用网络可用于优先考虑对肿瘤异质性更加稳定的治疗靶点。

## Results

### A ‘discovery and validation’ approach to the analysis of loss-of-function screens identifies reproducible genetic dependencies

We first wished to identify genetic interactions that could be independently reproduced across multiple distinct loss-of-function screens. To do this, we obtained gene sensitivity scores from four large-scale loss-of-function screens in panels of tumour cell lines, including two shRNA screens (DRIVE [McDonald et al., 2017], DEPMAP [Tsherniak et al., 2017]) and two CRISPR-Cas9 mutagenesis screens (AVANA [Meyers et al., 2017], SCORE [Behan et al., 2019]). We harmonised the cell line names across all studies, so they could be compared with each other and also with genotypic data (Barretina et al., 2012; Iorio et al., 2016; Figure 1A). In total, 917 tumour cell lines were screened in at least one loss-of-function study. Only 50 of these cell lines were common to all four studies while 407 cell lines were included in only a single study (Figure 1B). It is the partially overlapping nature of the screens that motivated the subsequent approach we took for our analysis. We used a ‘discovery set’ and ‘validation set’ approach to identifying genetic interactions across multiple screens - first identifying associations between driver gene alterations and gene inhibition sensitivity in the discovery study and then testing the discovered dependency in the validation study (Figure 1C). However, to ensure that any reproducibility observed was not merely due to cell lines common to both datasets, we first removed cell lines from the validation dataset if they were present in the discovery dataset (Figure 1C). For example, when using DEPMAP as the discovery dataset and AVANA as the validation dataset, we performed the validation analysis on the subset of cell lines that were present in AVANA but not in DEPMAP. In doing so, we ensured that any genetic interactions discovered were reproducible across different screening platforms (either distinct gene inhibition approaches, that is shRNA vs CRISPR, or distinct shRNA/CRISPR libraries) and also robust to the molecular heterogeneity seen across different cell line panels. Similar to our previous work (Bridgett et al., 2017; Campbell et al., 2016) we integrated copy number profiles and exome sequencing data to annotate all cell lines according to whether or not they featured likely functional alterations in any one of a panel of cancer driver genes (Vogelstein et al., 2013) (see Materials and methods, Supplementary file 1). We then identified associations between driver gene alterations and sensitivity to the inhibition of specific genes using a multiple regression model. This model included tissue type, microsatellite instability and driver gene status as independent variables and gene sensitivity score as the dependent variable (Materials and methods). Microsatellite instability was included as a covariate as it has previously been shown to be associated with non-driver gene specific dependencies (Behan et al., 2019), while tissue type was included to avoid confounding by tissue type. We focused this analysis on ‘selectively lethal’ genes - that is those genes whose inhibition killed some, but not all cell lines (Materials and methods, Supplementary file 2). We analysed each pair of screens in turn and considered a genetic dependency to be reproducible if it was validated in at least one discovery/validation pair. Using this approach, we tested 142,477 potential genetic dependencies between 61 driver genes and 2421 selectively lethal genes. We identified 1530 dependencies that were significant in at least one discovery screen (Figure 2A, Figure 2—figure supplement 1). All 61 driver genes had at least one dependency that was significant in at least one discovery screen while less than half of the selectively lethal genes (1,141/2,421) had a significant association with a driver gene. Of the 1530 dependencies that were significant in at least one discovery screen, only 229 could be validated in a second screen (Supplementary file 3, Figure 2A). For example, in the AVANA dataset TP53 mutation was associated with resistance to inhibition of both MDM4 and CENPF, but only the association with MDM4 could be validated in a second dataset (Figure 2B and C). Similarly, in the DEPMAP dataset NRAS mutation was associated with increased sensitivity to the inhibition of both NRAS itself and ERP44, but only the sensitivity to inhibition of NRAS could be validated in a second dataset (Figure 2B and C). The 229 reproducible dependencies involved 31 driver genes and 204 selectively lethal genes. Of the 229 reproducible genetic dependencies nine were ‘self vs. self’ associations, where the alteration of a gene was associated with sensitivity to its own inhibition. The majority (7/9) of these ‘self vs. self’ associations were oncogene addiction effects, such as the increased sensitivity of NRAS mutant cell lines to NRAS inhibition (Figure 2B). Similarly, we identified robust oncogene addictions involving the CTNNB1 (b-Catenin), KRAS, EGFR, BRAF, ERBB2 and PIK3CA oncogenes (Figure 3A and B, Figure 3—figure supplement 1B). For EGFR and CTNNB1, the only identified robust dependency was an oncogene addiction effect. For all other oncogenes there were additional robust dependencies identified, but in all cases the most significant reproducible dependency was an oncogene addiction (Figure 3—figure supplement 1A). These observations suggest that for most oncogenes the oncogene addiction effect might be more robust than any oncogene-related synthetic lethal effects. We also identified two (2/9) examples of ‘self vs. self’ dependencies involving tumour suppressors -TP53 (aka p53) and CDKN2A (aka p16/p14arf) (Figure 3—figure supplement 1C). This type of relationship has previously been reported for TP53: TP53 inhibition appears to offer a growth advantage to TP53 wild type cells but not to TP53 mutant cells (Giacomelli et al., 2018). Inhibiting TP53 in TP53 mutant cells has a largely neutral effect, while on average inhibiting TP53 in TP53 wild type cells actually increases fitness growth. Consequently, we observed an association between TP53 status and sensitivity to TP53 inhibition. Similar effects were seen for CDKN2A, although the growth increase resulting from inhibiting CDKN2A in wild-type cells is much lower than that seen for TP53 (Figure 3—figure supplement 1C). These ‘self vs. self’ dependencies, in particular the oncogene addictions, serve as evidence that our approach could identify well characterised genetic associations. However, as our primary interest was in genetic interactions between different genes, we excluded ‘self vs. self’ interactions from further analysis, leaving us with 220 robust genetic interactions (Figure 3C).

### Many robust genetic interactions reflect known pathway structure

Many of the robust genetic interactions we identified have been previously reported, including both sensitivity relationships, such as increased sensitivity of PTEN mutant cell lines to inhibition of the phosphoinositide 3-kinase-coding gene PIK3CB (Wee et al., 2008), and resistance relationships, such as an increased resistance of TP53 mutant cell lines to MDM2 inhibition (Figure 3D). Amongst the set of 220 robust genetic interactions, we identified two previously reported ‘paralog lethalities’ – synthetic lethal relationships between duplicate genes (paralogs) (Helming et al., 2014; Hoffman et al., 2014; Oike et al., 2013; Figure 3E). We found a robust association between mutation of the tumour suppressor ARID1A and sensitivity to inhibition of its paralog ARID1B (Helming et al., 2014) and also an association between mutation of SMARCA4 and sensitivity to inhibition of its paralog SMARCA2 (Hoffman et al., 2014; Oike et al., 2013). Both pairs of genes (ARID1A/ARID1B and SMARCA4/SMARCA2) encode components of the larger SWI/SNF complex (Wilson and Roberts, 2011). Some of the robust genetic dependencies could be readily interpreted using known pathway structures. For instance, many of the robust dependencies associated with the oncogene BRAF could be interpreted in terms of BRAF’s role in the MAPK pathway. BRAF mutation was associated with increased sensitivity to inhibition of its downstream effectors MEK (MAP2K1) and ERK (MAPK1), and increased resistance to inhibition of the alternative RAF isoform gene CRAF (RAF1) and the MAPK regulators PTPN11 and SHOC2 (Figure 4A and B; Hill et al., 2019). BRAF mutation was also associated with increased sensitivity to inhibition of PEA15, presumably a result of the requirement of PEA15 for ERK dimerisation and signalling activity (Formstecher et al., 2001; Herrero et al., 2015). Mutation or deletion of the tumour suppressor RB1 (Retinoblastoma 1, Rb) was associated with increased sensitivity or resistance to inhibition of multiple Rb pathway members (Figure 4C and D). We found that RB1 loss was reproducibly associated with resistance to inhibition of its negative regulators CDK4 and CDK6, consistent both with the known Rb pathway structure and with preclinical data suggesting that RB1 mutation confers resistance to CDK4/6 inhibitors (Asghar et al., 2015; O’Leary et al., 2018). Rb is a negative regulator of multiple E2F transcription factors, and we found that RB1 loss was reproducibly associated with increased sensitivity to both E2F1 and E2F3 inhibition (Figure 4C and D). RB1 loss was also associated with robust sensitivity to SKP2, a binding partner of Rb (Ji et al., 2004) first identified as an RB1 synthetic lethal partner in retinoblastoma (Xu et al., 2014) and more recently as a highly penetrant RB1 synthetic lethal partner in triple negative breast cancer (Brough et al., 2018; Figure 4C and D). Finally, RB1 loss was reproducibly associated with increased sensitivity to inhibition of Cyclin Dependent Kinase 2 (CDK2), suggesting that it may be a useful biomarker for CDK2-specific inhibitors (Tadesse et al., 2019). To test if this enrichment of pathway members among the robust dependencies associated with specific driver genes was a common phenomenon, for each driver gene with at least three dependencies we asked if these dependencies were enriched in specific signalling pathways (see Materials and methods). Of the twelve driver genes tested, we found that five of these were enriched in specific pathways and in all five cases found that the driver gene itself was also annotated as a member of the most enriched pathway (Supplementary file 4). As expected RB1 (most enriched pathway ‘G1 Phase’) and BRAF (most enriched pathway ‘Negative feedback regulation of MAPK pathway’) were among the five driver genes, alongside PTEN (‘PI3K/AKT activation’), CDKN2A (‘Cell cycle’), and NRAS (‘Ras signaling pathway’).

我们发现的许多稳健的遗传相互作用以前都有报道，包括敏感性关系，如PTEN突变体细胞系对抑制磷酸肌苷3-激酶编码基因PIK3CB的敏感性增加（Wee等，2008），以及抗性关系，如TP53突变体细胞系对MDM2抑制的抗性增加（图3D）。在这组220个稳健的遗传相互作用中，我们发现了两个先前报道的 "类属致死"--重复基因（类属）之间的合成致死关系（Helming等人，2014；Hoffman等人，2014；Oike等人，2013；图3E）。我们发现肿瘤抑制因子ARID1A的突变与对其同族体ARID1B的抑制的敏感性之间存在强有力的关联（Helming等人，2014），也发现SMARCA4的突变与对其同族体SMARCA2的抑制的敏感性之间存在关联（Hoffman等人，2014；Oike等人，2013）。这两对基因（ARID1A/ARID1B和SMARCA4/SMARCA2）都编码较大的SWI/SNF复合物的成分（Wilson和Roberts，2011）。一些强大的遗传依赖性可以很容易地用已知的路径结构进行解释。例如，许多与致癌基因BRAF有关的强大依赖性可以用BRAF在MAPK途径中的作用来解释。BRAF突变与对其下游效应物MEK（MAP2K1）和ERK（MAPK1）的抑制敏感性增加有关，而对替代RAF异构体基因CRAF（RAF1）和MAPK调节器PTPN11和SHOC2的抑制抗性增加（图4A和B；Hill等人，2019）。BRAF突变也与对PEA15抑制的敏感性增加有关，可能是PEA15对ERK二聚化和信号活动的要求的结果（Formstecher等，2001；Herrero等，2015）。肿瘤抑制因子RB1（视网膜母细胞瘤1，Rb）的突变或缺失与对多个Rb通路成员的抑制的敏感性或抗性增加有关（图4C和D）。我们发现，RB1的缺失与对其负调控因子CDK4和CDK6的抑制的耐药性有重复性的关系，这与已知的Rb通路结构和临床前数据一致，表明RB1突变会赋予CDK4/6抑制剂以耐药性（Asghar等人，2015；O'Leary等人，2018）。Rb是多个E2F转录因子的负向调节器，我们发现RB1的缺失与E2F1和E2F3抑制的敏感性增加重复相关（图4C和D）。RB1缺失还与对SKP2的敏感性有关，SKP2是Rb的结合伙伴（Ji等人，2004年），首次被确定为视网膜母细胞瘤中的RB1合成致死伙伴（Xu等人，2014年），最近又被确定为三阴性乳腺癌中高度渗透的RB1合成致死伙伴（Brough等人，2018年；图4C和D）。最后，RB1的丢失与对细胞周期蛋白依赖性激酶2（CDK2）抑制的敏感性增加有重复性的关系，这表明它可能是CDK2特异性抑制剂的一个有用的生物标志物（Tadesse等人，2019）。为了测试这种与特定驱动基因相关的强大依赖性中的通路成员的富集是否是一种普遍现象，对于每个具有至少三个依赖性的驱动基因，我们询问这些依赖性是否富集于特定的信号通路（见材料和方法）。在测试的12个驱动基因中，我们发现其中有5个富集在特定的通路中，并且在所有5个案例中发现驱动基因本身也被注释为最富集通路的成员（补充文件4）。不出所料，RB1（最富集的途径 "G1阶段"）和BRAF（最富集的途径 "MAPK途径的负反馈调节"）与PTEN（"PI3K/AKT激活"）、CDKN2A（"细胞周期"）和NRAS（"Ras信号途径"）一起成为这五个驱动基因。

### Robust genetic interactions are enriched in protein–protein interaction pairs

In seeking to understand what particular characteristics robust genetic interactions might have, we noted that many of the robust genetic interactions we identified involved gene pairs whose protein products operate in the same pathway (e.g. the Rb pathway) or protein complex (e.g. SWI/SNF) suggesting that genetic interactions between gene pairs whose protein products physically interact may be more robust than other genetic interactions. To test this hypothesis, we compared the robust genetic interactions we identified with protein–protein interactions from the STRING protein–protein interaction database (Szklarczyk et al., 2015). We found that, when considering the set of all gene pairs tested, gene pairs identified as significant genetic interactors in at least one dataset are more likely to encode proteins that physically interact (Figure 5A) (Odds Ratio (OR) = 4.0, p<210 16, Fisher’s Exact test). Furthermore, of the genetic interactions identified as significant in at least one dataset, those that are supported by a protein–protein interaction were significantly more likely to be reproduced in a second dataset (Figure 5A) (OR = 3.9, p<110 13). We therefore concluded that protein–protein interaction pairs are more likely to be significant hits in one dataset and even more likely to be reproduced across multiple datasets, suggesting this might be a feature of robust genetic interaction effects. We noted that a large number (n = 132) of robust genetic interactions involved TP53, presumably as a result of the high number of TP53 mutant tumour cell lines in the datasets (and its high mutation frequency in human cancer) and the associated increased statistical power to detect TP53-related genetic interactions. We therefore considered whether the significant number of TP53-related genetic interactions in our dataset could confound our analyses, especially as TP53 is also associated with a disproportionately high number of protein interactions (>1700 medium confidence interactors in the STRING database alone, compared to a median of 37 medium confidence interactions across all proteins). However, even after excluding genetic interactions involving TP53, the observation that robust genetic interactions were enriched in protein–protein interaction pairs was still evident (Figure 5B); known protein interaction pairs were more likely to be identified as significant genetic interactions in one screen (OR = 3.8, p<210 16) and among the significant genetic interactions discovered in one screen, those involving protein–protein interaction pairs were more likely to be reproduced in a validation screen (OR = 9.3, p<210 16). The same effects were observed when considering genetic interactions observed at different false-discovery rate (FDR) thresholds (Figure 5—figure supplement 1A,B) and using different sources of protein–protein interaction data (Figure 5—figure supplement 1C,D,E, Supplementary file 5; Alanis-Lobato et al., 2017; ChatrAryamontri et al., 2015; Huttlin et al., 2020). Previous work has demonstrated that the protein–protein interaction networks aggregated in databases are subject to significant ascertainment bias – some genes are more widely studied than others and this can result in them having more reported protein–protein interaction partners than other genes (Rolland et al., 2014). As cancer driver genes are studied more widely than most genes, they may be especially subject to this bias. To ensure the observed enrichment of protein–protein interactions among genetically interacting pairs was not simply due to this ascertainment bias, we compared the results observed for the real STRING protein–protein interaction network with 100 degree-matched randomised networks and again found that there was a higher than expected overlap between protein–protein interactions and both discovered and validated genetic interactions (Figure 5—figure supplement 2). The increased reproducibility of genetic interactions associated with protein–protein interactions across different genetic perturbation screen datasets could have two distinct causes - increased reproducibility across distinct technologies or libraries (e.g. CRISPR/shRNA) or increased reproducibility/robustness of genetic interactions in cell line panels with distinct molecular backgrounds. To test the former possibility, we repeated our discovery/validation approach but focused on the set of cell lines that were common to different genetic perturbation screen datasets. Using this approach, the molecular backgrounds (i.e. cell lines) tested were the same, but the screening approach or library used differed. Upon doing this, we found that genetic interactions between gene pairs whose protein products physically interact were significantly more reproducible across studies (Figure 5C, OR = 6.1 and p<210 10 when compared to discovered genetic interactions) (Supplementary file 5). To test reproducibility using the same screening approach across molecularly distinct cell lines, we artificially split individual datasets into non-overlapping discovery and validation sets of cell lines. Again, we found that genetic interactions between gene pairs whose protein products physically interact were more reproducible across distinct cell line panels (Figure 5D, OR = 8.0 and p<110 12 when compared to discovered genetic interactions) (Supplementary file 5). We therefore concluded that genetic interactions supported by protein–protein interactions were more reproducible across different screening approaches and across distinct cell line contexts, suggesting that these interactions are, overall, more robust.



##强大的遗传相互作用在蛋白质-蛋白质相互作用对中得到了富集

在寻求了解稳健的遗传相互作用可能具有的特定特征时，我们注意到，我们确定的许多稳健的遗传相互作用涉及其蛋白产品在同一途径（如Rb途径）或蛋白复合物（如SWI/SNF）中运作的基因对，这表明其蛋白产品物理相互作用的基因对之间的遗传相互作用可能比其他遗传相互作用更稳健。为了验证这一假设，我们将我们确定的稳健的遗传相互作用与STRING蛋白质-蛋白质相互作用数据库（Szklarczyk等人，2015）中的蛋白质-蛋白质相互作用进行了比较。我们发现，当考虑到所有测试的基因对集合时，在至少一个数据集中被确定为重要的遗传相互作用的基因对更有可能编码物理相互作用的蛋白质（图5A）（Odds Ratio（OR）=4.0，P<210 16，Fisher's Exact检验）。此外，在至少一个数据集中被确认为显著的遗传相互作用中，那些由蛋白质-蛋白质相互作用支持的遗传相互作用在第二个数据集中被重现的可能性明显更大（图5A）（OR=3.9，P<110 13）。因此，我们得出结论，蛋白质-蛋白质相互作用对更有可能在一个数据集中成为重要的命题，甚至更有可能在多个数据集中重现，表明这可能是稳健遗传相互作用效应的一个特征。我们注意到，大量（n = 132）稳健的遗传相互作用涉及TP53，这可能是由于数据集中有大量的TP53突变肿瘤细胞系（以及其在人类癌症中的高突变频率），以及相关的检测TP53相关遗传相互作用的统计能力提高。因此，我们考虑了我们的数据集中大量的TP53相关的遗传相互作用是否会混淆我们的分析，特别是TP53也与不成比例的高数量的蛋白质相互作用有关（仅在STRING数据库中就有>1700个中等可信度的相互作用，而所有蛋白质的中等可信度相互作用的中位数为37）。然而，即使排除了涉及TP53的遗传相互作用，稳健的遗传相互作用在蛋白质-蛋白质相互作用对中富集的观察仍然是明显的（图5B）；已知的蛋白质相互作用对更有可能在一次筛选中被确定为重要的遗传相互作用（OR=3.8，p<210 16），在一次筛选中发现的重要遗传相互作用中，涉及蛋白质-蛋白质相互作用对的相互作用更有可能在验证筛选中被再现（OR=9.3，p<210 16）。考虑到在不同的假发现率（FDR）阈值下观察到的遗传相互作用（图5-图补充1A,B）和使用不同来源的蛋白-蛋白相互作用数据（图5-图补充1C,D,E，补充文件5；Alanis-Lobato等，2017；ChatrAryamontri等，2015；Huttlin等，2020）时，也观察到同样的效果。以前的工作已经证明，数据库中聚集的蛋白质-蛋白质相互作用网络存在明显的确认偏差--一些基因比其他基因被更广泛地研究，这可能导致它们比其他基因有更多报告的蛋白质-蛋白质相互作用伙伴（Rolland等人，2014）。由于癌症驱动基因的研究比大多数基因更广泛，它们可能特别受到这种偏见的影响。为了确保观察到的蛋白质-蛋白质相互作用在基因相互作用对中的富集不只是由于这种确认偏差，我们将真实的STRING蛋白质-蛋白质相互作用网络与100度匹配的随机网络观察到的结果进行了比较，再次发现蛋白质-蛋白质相互作用与发现和验证的基因相互作用之间的重合度高于预期（图5-图补充2）。在不同的遗传扰动筛选数据集中，与蛋白质-蛋白质相互作用相关的遗传相互作用的可重复性增加，可能有两个不同的原因--在不同的技术或文库（如CRISPR/shRNA）中可重复性增加，或者在具有不同分子背景的细胞系板中遗传相互作用的可重复性/稳健性增加。为了测试前一种可能性，我们重复了我们的发现/验证方法，但把重点放在不同遗传扰动筛选数据集所共有的细胞系上。使用这种方法，测试的分子背景（即细胞系）是相同的，但使用的筛选方法或库是不同的。这样做后，我们发现，其蛋白质产物有物理相互作用的基因对之间的遗传相互作用在不同的研究中明显更有可重复性（图5C，与发现的遗传相互作用相比，OR=6.1，P<210 10）（补充文件5）。为了测试使用相同的筛选方法在分子不同的细胞系中的可重复性，我们人为地将单个数据集分成不重叠的发现和验证的细胞系集。同样，我们发现，在不同的细胞系面板上，其蛋白产物发生物理相互作用的基因对之间的遗传相互作用更具有可重复性（图5D，与发现的遗传相互作用相比，OR=8.0，P<110 12）（补充文件5）。因此，我们得出结论，由蛋白质-蛋白质相互作用支持的遗传相互作用在不同的筛选方法和不同的细胞系背景下更具有可重复性，这表明这些相互作用在总体上是更强大的。





### Prioritising robust synthetic lethal interactions involving passenger gene alterations

In addition to alterations of cancer driver genes, tumour cells typically harbour genetic alterations of large numbers of ‘passenger’ genes. Although these genes may not facilitate tumourigenesis or promote cancer cell growth, their alteration may still impart genetic vulnerabilities upon tumour cells. Indeed, multiple synthetic lethal interactions have been identified involving passenger genes that exhibit recurrent copy number loss in cancer cells due to their chromosomal proximity to tumour suppressor genes lost via loss-of-heterozygosity (Kryukov et al., 2016; Marjon et al., 2016; Mavrakis et al., 2016; Muller et al., 2012; Muller et al., 2015). The space of genetic interactions to test involving passenger gene alterations is much larger than that involving driver genes, as nearly every gene in the genome is either mutated or deleted in some cancer context. In addition, passenger genes are typically altered at frequencies lower than for driver genes and therefore the statistical power to identify genetic interactions associated with their alteration is somewhat reduced. With these issues in mind, we reasoned that protein–protein interaction maps might help narrow the search space considerably and thus reduce the burden of multiple hypothesis testing. For all passenger genes that were recurrently lost in at least ten tumour cell lines, either through homozygous deletion or loss-of-function mutation, we searched for genetic interactions with their protein–protein interaction partners using the same discovery and validation approach previously used for driver genes. In total we tested 47,781 interactions involving 2,972 passenger genes and 2149 selectively lethal genes. To perform an all-against-all test without filtering based on protein–protein interactions would have required more than six million tests, significantly increasing the burden of multiplehypothesis testing. At an FDR of 20% we found 11 robust genetic interactions involving passenger gene alterations (Supplementary file 7). To assess whether this is more than would be expected by chance we randomly sampled 47,781 gene pairs from the same search space 100 times. The median number of robust genetic interactions identified amongst these randomly sampled gene pairs was 1 (mean 1.27, min 0, max 6) suggesting that the 11 robust genetic interactions observed among protein–protein interacting pairs was more than would be expected by chance. Three of the robust interactions involve genes frequently deleted with the tumour suppressor CDKN2A (CDKN2B and MTAP) and recapitulate known associations with CDKN2A. A further two genetic interactions involve a single chromosomal region (19p21.3) containing two interferon genes (IFNB1 and IFNW1) which are frequently deleted together and consequently these two interactions really represent a single association (an increased sensitivity to thrombopoietin receptor MPL). Of the six remaining genetic interactions identified, four represent examples of paralog lethalities – loss of one member of a paralog pair is associated with increased sensitivity to the inhibition of the other member. RPL22 loss was associated with increased sensitivity to its paralog RPL22L1, TIMM17B with its paralog TIMM17A, DDX17 with its paralog DDX5, and VPS4B with its paralog VPS4A. We selected two of these robust synthetic lethal interactions for further validation – VPS4B/VPS4A and DDX17/DDX5. VPS4A and VPS4B are highly sequence similar whole genome duplicates with protein sequence identity of 81%. Both proteins can form a complex with the Vacuolar protein sorting-associated protein VTA1 (Huttlin et al., 2017) and are involved in endosomal trafficking. VPS4B is located at 18q21 and is frequently deleted with the tumour suppressor SMAD4, explaining the relatively high frequency of loss of VPS4B in cancer. Previous analysis of the DRIVE shRNA dataset identified an association between VPS4B copy number loss and VPS4A sensitivity (McDonald et al., 2017). Here we find evidence of this association in two additional datasets – AVANA and SCORE (Figure 7A). Although this association is robust, it does not establish a causal link between VPS4B loss and VPS4A sensitivity. Indeed, there are 39 protein-coding genes on chromosome 18 located between SMAD4 and VPS4B, any one of which could cause sensitivity to VPS4A inhibition. To verify that VPS4B is the cause of VPS4A sensitivity we transfected isogenic knockouts of either VPS4A and VPS4B with siRNA smartpools targeting either VPS4A or VPS4B and found that, consistent with a negative genetic interaction between the two genes, compared to wildtype parental cells VPS4A knockout cells were more sensitive to siRNA targeting VPS4B and VPS4B knockout cells were more sensitive to siRNA targeting VPS4A (Figure 7B, Supplementary file 8). Like VPS4A and VPS4B, DDX5 and DDX17 are widely conserved highly sequence similar whole genome duplicates (protein sequence identity 69%). They are DEAD box family RNA helicases that have multiple roles in both transcription and splicing; they act as coregulators for multiple transcription factors and also function as components of the spliceosome (Dardenne et al., 2014; FullerPace, 2013). A direct protein–protein interaction between the two genes has also been reported (Hegele et al., 2012; Huttlin et al., 2017). DDX17 is located at 22q12 in close proximity to the tumour suppressor MYH9, potentially explaining its recurrent deletion in tumour cell lines. We identified an association between DDX17 loss and DDX5 sensitivity in the AVANA CRISPR dataset and validated this association in the DRIVE shRNA dataset (Figure 7C). As with VPS4A/VPS4B, to verify that DDX17 loss is the cause of DDX5 sensitivity we transfected isogenic knockouts of either DDX17 and DDX5 with siRNA smartpools targeting either DDX5 or DDX17. We found that, consistent with a negative genetic interaction between the two genes, compared to wildtype parental cells DDX17 knockout cells were more sensitive to siRNA targeting DDX5 and DDX5 knockout cells were more sensitive to siRNA targeting DDX17 (Figure 7D, Supplementary file 8).

## Discussion

While the reproducibility of pharmacogenomic screens in cancer cell lines has been much discussed (Cancer Cell Line Encyclopedia Consortium and Genomics of Drug Sensitivity in Cancer Consortium, 2015; Haibe-Kains et al., 2013; Niepel et al., 2019), relatively little attention has been paid to the reproducibility of results from large-scale genetic screens in cell lines. Analyses of the pharmacogenomic screen datasets have primarily focused on reproducibility in a very strict sense - that is quantifying the extent to which the same drug elicits the same response in the same cell line when assayed across different sites (Niepel et al., 2019). In some cases these analyses have been extended to quantify the extent to which the same associations between biomarkers and drugs can be observed across the same cell line panels assayed in different experiments (Cancer Cell Line Encyclopedia Consortium and Genomics of Drug Sensitivity in Cancer Consortium, 2015). Here we were interested in reproducibility in a much broader sense and sought to identify genetic interactions that could be reproduced both across distinct experiments and across distinct cell line panels, that is interactions that are robust to genetic and molecular heterogeneity. We developed an approach to identify these robust genetic interactions and used it to identify a set of 220 robust genetic interactions associated with cancer driver genes. We found that these robust genetic interactions are enriched among gene pairs whose protein products physically interact, suggesting a means by which we might prioritise the most promising candidates from screens for follow on studies. We do not claim that our set of robust genetic interactions is comprehensive, as there are many reasons why real robust genetic interactions may not be identified by our approach. There are many driver genes that we have not included in our analysis because they are infrequently mutated in the datasets studied. Consequently, we can report no interactions for these genes. We have also focussed only on identifying interactions associated with mutation or copy number changes. There are likely to be dependencies associated with altered gene/protein expression that will be missed by this approach. Furthermore, for the genes that we do analyse, it is likely that some real interactions are not detected due to a lack of statistical power. Finally, of the dependencies identified in a discovery screen but absent in a validation screen, false negatives due to reagents with poor gene targeting ability likely play a significant role (Kaelin, 2012). We have exclusively focussed on identifying dependencies that are evident across panels of cell lines from multiple cancer types (‘pan-cancer dependencies’). It is likely that there are robust dependencies only evident within specific cancer types, but it is difficult to use our approach to identify them due to the restricted number of cell lines available for each cancer type. Even with a relatively common mutation (e.g. KRAS mutation in non-small cell lung cancer) it is challenging to partition the available cell lines into distinct discovery and validation sets while maintaining statistical power to identify potential dependencies. This issue may be alleviated by efforts to create large numbers of new tumour cell lines (Boehm and Golub, 2015) or through using isogenic models for discovery and cell line panels for validation (Ryan et al., 2018). Many published synthetic lethal screens have focussed on identifying new drug targets for ‘undruggable’ oncogenes such as MYC and RAS (reviewed in Cermelli et al., 2014 and Downward, 2015 respectively). The rationale for such studies is that the oncogene addiction itself cannot be exploited directly and consequently a synthetic lethal approach is needed. However, here we found that for all oncogenes studied the most significant reproducible dependency identified was an oncogene addiction (Figure 3A). This suggests that any synthetic lethal interactions that are identified for oncogenes will likely be of a smaller effect size or operate in a more restricted context than the oncogene addiction itself. Previous work has suggested this to be true of KRAS (McDonald et al., 2017) but here we find that it appears to be a general property of all oncogenes studied. This implies that wherever possible, direct targeting of oncogenes might be more therapeutically effective than exploiting oncogene-related synthetic lethal effects. Our approach to identify robust genetic dependencies involving cancer driver genes is unbiased in the sense that we did not incorporate prior knowledge of functional relationships to identify candidate gene pairs to test. Nonetheless, many of the robust synthetic lethalities identified reflect known biology. In particular, for each of the well-studied tumour suppressors ARID1A, SMARCA4 and PTEN the most significant robust synthetic lethal interaction we identified has previously been reported in the literature. For ARID1A, its known synthetic lethal partner ARID1B was the only robust candidate interaction identified while for PTEN and SMARCA4 their established synthetic lethal partners (PIK3CB and SMARCA2 respectively) are the most significant robust hits by a large margin (Figure 3C). As with oncogenes, this suggests that if novel single gene vulnerabilities for these drivers are to be discovered, they may have a smaller effect size or operate in a more restricted setting. In budding and fission yeast, multiple studies have shown that genetic interactions are enriched among protein–protein interaction pairs and vice-versa (Costanzo et al., 2010; Kelley and Ideker, 2005; Michaut et al., 2011; Roguev et al., 2008). Pairwise genetic interaction screens in individual mammalian cell lines have also revealed an enrichment of genetic interactions among protein–protein interaction pairs (Han et al., 2017; Roguev et al., 2013). Our observation that discovered genetic interactions are enriched in protein–protein interaction pairs is consistent with these studies. However, these studies have not revealed what factors influence the conservation of genetic interactions across distinct genetic backgrounds, that is what predicts the robustness of a genetic interaction. In yeast, the genetic interaction mapping approach relies on mating gene deletion mutants and consequently the vast majority of reported genetic interactions are observed in a single genetic background (Tong et al., 2001). In mammalian cells, pairwise genetic interaction screens across multiple cell lines have revealed differences across cell lines but not identified what factors influence the conservation of genetic interactions across cell lines (Shen et al., 2017). While variation of genetic interactions across different strains or different genetic backgrounds has been poorly studied, previous work has analysed the conservation of genetic interactions across species and shown that genetic interactions between gene pairs whose protein products physically interact are more highly conserved (Roguev et al., 2008; Ryan et al., 2012; Srivas et al., 2016). Our analysis here suggests that the same principles may be used to identify genetic interactions conserved across genetically heterogeneous tumour cell lines. Previous work has also shown that genetic interactions between gene pairs involved in the same biological process, as indicated by annotation to the same gene ontology term, are more highly conserved across species (Ryan et al., 2012; Srivas et al., 2016). Similarly, genetic interactions that are stable across experimental conditions (e.g. in the presence and absence of different DNA damaging agents) are more likely to be conserved across species (Srivas et al., 2016). Although we have not tested them here, these additional features predictive of between-species conservation may also be predictive of robustness to genetic heterogeneity. Our set of robust genetic interactions may serve as the starting point for such analyses and may also serve as a training set for computational approaches to predict synthetic lethality (JerbyArnon et al., 2014). Our finding that the robust genetic interactions associated with some driver genes can be interpreted in terms of the signalling pathway that the driver gene functions in suggests that pathway structure may also provide information on robustness. For example, it seems reasonable to hypothesise that synthetic lethal proteins that are in close vicinity in a pathway are more likely to exhibit a robust synthetic lethal than those that are more distantly connected. However, to allow such a hypothesis to be tested, we believe the annotation of molecular pathways should be somewhat more reliable and the set of experimentally-validated robust genetic interactions much larger. Our results suggest that knowledge of protein–protein interactions could be used to improve the design and analysis of loss-of-function screens for synthetic lethal interactions. We have demonstrated the utility of incorporating such prior knowledge for identifying robust synthetic lethal interactions from genome-wide chemogenetic screens. We have also demonstrated that protein–protein interactions can aid the identification of genetic interactions associated with passenger gene alterations, where statistical power is limited due to relatively infrequent alterations and the number of potential interactions to test is enormous. An alternative to these approaches, where knowledge of protein–protein interactions is used after the screen has already been performed, would be to screen target libraries for specific driver genes based on their known protein interaction partners. Regardless of the approach used to identify candidate synthetic lethal interactions in a large-scale screen, our results suggest that candidates supported by a protein–protein interaction should be prioritised for follow on study as they are more likely to be robust to the genetic heterogeneity observed in tumour cells.

### 结果

#### 一种“发现和验证”方法用于分析功能丧失筛选，识别可重复的基因依赖关系

我们首先希望识别可以在多个不同的功能丧失筛选中独立重复的基因相互作用。为此，我们从四个大规模功能丧失筛选中获取基因敏感性分数，包括两个shRNA筛选（DRIVE [McDonald et al., 2017]，DEPMAP [Tsherniak et al., 2017]）和两个CRISPR-Cas9突变筛选（AVANA [Meyers et al., 2017]，SCORE [Behan et al., 2019]）。我们对所有研究进行了细胞系命名的协调，以便彼此之间和与基因型数据（Barretina et al., 2012; Iorio et al., 2016）进行比较（图1A）。总共，在至少一个功能丧失研究中筛选了917个肿瘤细胞系。其中只有50个细胞系在所有四个研究中共同存在，而407个细胞系仅包含在单个研究中（图1B）。由于筛选的部分重叠性质，我们采用了“发现集”和“验证集”的方法来识别多个筛选中的基因相互作用。首先，在发现研究中识别驱动基因突变与基因抑制敏感性之间的关联，然后在验证研究中测试发现的依赖关系（图1C）。但为了确保观察到的可重复性不仅仅是由于在发现数据集和验证数据集中都存在的细胞系，我们首先从验证数据集中删除了存在于发现数据集中的细胞系（图1C）。例如，当使用DEPMAP作为发现数据集，AVANA作为验证数据集时，我们在AVANA数据集中但不在DEPMAP数据集中的细胞系子集上进行验证分析。通过这样做，我们确保了任何发现的基因相互作用都能够在不同的筛选平台（即不同的基因抑制方法，如shRNA vs CRISPR）和不同的细胞系组间的分子异质性面前重复。与我们以前的工作类似（Bridgett et al., 2017; Campbell et al., 2016），我们使用拷贝数配置文件和外显子测序数据，根据细胞系是否具有一组癌症驱动基因中的潜在功能性改变对所有细胞系进行注释（Vogelstein et al., 2013）（详见材料和方法，补充文件1）。然后，我们使用多元回归模型来识别驱动基因突变与特定基因抑制敏感性之间的关联。该模型包括组织类型、微卫星不稳定性和驱动基因状态作为独立变量，基因敏感性得分作为因变量（材料和方法）。微卫星不稳定性被包括为协变量，因为之前已经发现它与非驱动基因特异性依赖关系相关（Behan et al., 2019），而组织类型被包括以避免组织类型混淆。我们将重点放在“选择性致死”基因的分析上，即那些抑制某些细胞系但不是所有细胞系的基因（材料和方法，补充文件2）。我们逐一分析了每对筛选，并考虑一个基因依赖关系在至少一个发现/验证对中得到验证时才被认为是可重复的。采用这种方法，我们测试了61个驱动基因和2421个选择性致死基因之间的142,477个潜在基因依赖关系。我们发现在至少一个发现筛选中有1530个依赖关系是显著的（图2A，图2—补充资料1）。所有61个驱动基因至少有一个依赖关系在至少一个发现筛选中是显著的，而只有不到一半的选择性致死基因（1,141/2,421）与驱动基因的显著关联。在在至少一个发现筛选中显著的1530个依赖关系中，只有229个可以在第二个筛选中得到验证（补充文件3，图2A）。例如，在AVANA数据集中，TP53突变与MDM4和CENPF抑制的耐药性都有关联，但只有与MDM4的关联可以在第二个数据集中得到验证（图2B和C）。类似地，在DEPMAP数据集中，NRAS突变与NRAS本身和ERP44的抑制敏感性增加有关，但只有NRAS的敏感性抑制在第二个数据集中得到验证（图2B和C）。这229个可重复的依赖关系涉及31个驱动基因和204个选择性致死基因。在这229个可重复的基因依赖关系中，有9个是“自我对自我”的关联，即基因的改变与对其自身的抑制敏感性有关。其中大多数（7/9）是癌基因成瘾效应（Supplementary File 4）。总的来说，我们的结果表明，虽然在不同的功能丧失筛选中存在很多基因相互作用，但只有少数关联是可重复的。

#### 基于多个数据集的交叉验证提供了更多的基因依赖关系验证

我们进一步验证了我们发现的基因相互作用是否能够在不同的数据集和研究中得到验证。为此，我们从两个不同的细胞系依赖数据集中获取了独立的验证数据，即CCLE [Barretina et al., 2012]和GDSC [Iorio et al., 2016]。这两个数据集使用了不同的基因抑制方法（shRNA和CRISPR）和细胞系集合，并且研究目的不同于前面的筛选（图3A）。在CCLE数据集中，我们使用CRISPR基因编辑数据（Hart et al., 2015）来验证在发现和验证研究中发现的基因依赖关系。在GDSC数据集中，我们使用shRNA筛选数据（Barretina et al., 2012）进行验证。我们首先检查了在这两个数据集中是否存在相应的基因抑制数据以验证发现的依赖关系。然后，我们使用与之前相同的多元回归模型来识别在这两个独立数据集中的基因依赖关系。我们只考虑在发现和验证对中都有显著关联的依赖关系，并且在两个独立数据集中都能够得到验证。这种交叉验证方法可以提供对我们发现的基因相互作用的额外验证。在CCLE数据集中，我们发现有26个基因依赖关系可以在验证数据集中得到验证（图3B）。同样，在GDSC数据集中，我们发现有19个基因依赖关系可以在验证数据集中得到验证（图3C）。总的来说，在CCLE和GDSC数据集的交叉验证中，我们发现了45个可重复的基因依赖关系。这些结果进一步加强了我们发现的基因相互作用的可靠性和重要性。

#### 功能丧失筛选揭示了新的基因相互作用和细胞适应机制

通过分析功能丧失筛选数据，我们不仅能够验证已知的基因相互作用，还能够发现新的基因相互作用和细胞适应机制。我们发现了一些与特定驱动基因突变有关的新的基因依赖关系。例如，在发现筛选中，我们发现了一个与TP53突变相关的基因依赖关系，即MDM4抑制敏感性的增加。这表明在具有TP53突变的细胞系中，抑制MDM4可能是一种潜在的治疗策略。我们还发现了一些与NRAS突变相关的基因依赖关系，即NRAS抑制敏感性的增加，以及与ERP44抑制敏感性的增加有关。这些结果提供了进一步的证据支持在具有NRAS突变的细胞系中针对NRAS和ERP44的治疗靶点。另外，我们还发现了一些自我对自我的关联，即基因的突变与对其自身的抑制敏感性有关。这些结果提供了关于基因自发性失活机制的新见解。总体而言，通过分析功能丧失筛选数据，我们能够鉴定和验证新的基因相互作用和细胞适应机制。

#### 结论

在本研究中，我们使用一种“发现和验证”的方法来分析多个功能丧失筛选数据，识别可重复的基因依赖关系。我们发现只有少数基因相互作用在多个筛选中可重复，表明功能丧失筛选的结果存在一定的异质性。然而，通过基于多个数据集的交叉验证，我们能够进一步验证发现的基因依赖关系，加强了它们的可靠性和重要性。通过分析功能丧失筛选数据，我们不仅能够验证已知的基因相互作用，还能够发现新的基因相互作用和细胞适应机制。这些结果对于理解基因调控网络和癌症治疗的发展具有重要意义。

## Materials and methods

## Loss of function screens

Different scoring systems have been developed for calculating ‘gene level’ sensitivity scores from loss-of-function screens performed with multiple gene targeting reagents per gene (i.e. shRNAs or gRNAs). For the analysis of all loss-of-function screens we used the original authors’ own preferred approaches. CERES sensitivity scores (Meyers et al., 2017) for AVANA (release 18Q4) were obtained from the DepMap portal (https://depmap.org/portal/download/) (RRID:SCR_017655), while DEMETER v2 gene sensitivity scores for the DEPMAP shRNA screen (Tsherniak et al., 2017) were obtained from the same resource. For the DEPMAP screen, some genes were only screened in a subset of cell lines and these were excluded from all analyses. Quantile normalized CRISPRcleaned (Iorio et al., 2018) depletion log fold changes for Project SCORE (Behan et al., 2019) were obtained from the Project SCORE database (https://score.depmap.sanger.ac.uk/). ATARIS (Shao et al., 2013) scores for the DRIVE dataset (McDonald et al., 2017) were obtained from the authors. 28 of the 398 cell lines screened in DRIVE had missing gene scores for ~25% of genes screened and these cell lines were excluded from further analysis. All screens were mapped to a common cell line name format (that followed by the Cancer Cell Line Encyclopaedia [Barretina et al., 2012]) using the Cell Model Passports resource where appropriate (van der Meer et al., 2019).

### Identifying selectively lethal genes

Similar to previous work (McDonald et al., 2017; Tsherniak et al., 2017), to reduce the burden of multiple hypothesis testing we focused our analysis on genes whose inhibition appeared to cause growth defects in subsets of the cancer cell lines screened. That is, rather than testing for associations with genes whose inhibition was always lethal or never lethal, we focused our analyses on genes that could be associated with distinct sensitive and resistant cell line cohorts. We first identified a set of ‘selectively lethal’ genes using the AVANA dataset (Meyers et al., 2017) - those with a gene sensitivity score < 0.6 in at least 10 cell lines but no more than 259 cell lines (half of the screened cell lines). We augmented this with a list of 65 ‘outlier genes’ identified by the authors of the DRIVE study as having a skewed distribution suggesting distinct sensitive and resistant cohorts (McDonald et al., 2017). Finally from the combined list we removed genes known to be commonly essential in cancer cell lines (Hart et al., 2017). This resulted in a set of 2470 selectively lethal genes (Supplementary file 2) which were used for all association analyses.

### Identifying gene alterations from copy number and exome profiling

For all cell lines we obtained sequencing data (CCLE_DepMap_18q3_maf_20180718.txt) and copy number profiles (public_18Q3_gene_cn_v2.csv) from the DepMap portal. These datasets contain integrated genotyping data from both the Cancer Cell Line Encyclopedia and GDSC resources (Barretina et al., 2012; Cancer Cell Line Encyclopedia Consortium and Genomics of Drug Sensitivity in Cancer Consortium, 2015; Iorio et al., 2016). We used this to identify likely functional alterations in a panel of cancer driver genes (Vogelstein et al., 2013) restricting our analysis to those genes that were subject to targeted sequencing as part of the Cancer Cell Line Encyclopedia (Barretina et al., 2012). For most oncogenes we considered the gene to be functionally altered if it contained a protein altering mutation at a residue that is recurrently altered in either the COSMIC database or the Cancer Genome Atlas. For a small number of oncogenes (ERBB2, CCND1, MDM2, MDM4) we considered them to be functionally altered only if they were amplified. For all tumour suppressors we considered all protein-coding mutations and homozygous deletions to be functional alterations. We used a threshold of 1.28 to call homozygous deletions. The matrix of functional alterations to driver genes is presented in Supplementary file 1. To identify loss-of-function alterations to passenger genes, a similar pipeline was used. However the driver genes from Vogelstein et al., 2013 were excluded from analysis and only clear loss-offunction alterations were considered to be functional. The matrix of gene loss in passenger genes is presented in Supplementary file 6.

### Identifying genetic dependencies in individual datasets

We wished to identify associations between driver gene mutations and gene sensitivity scores that were not confounded by tissue specific gene sensitivity effects (e.g. SOX10 sensitivity scores can be naively associated with BRAF mutational status because SOX10 is essential in melanoma cell lines and BRAF mutation is common in melanoma). Thus, we wished to model gene sensitivity after first accounting for tissue type. To this end, associations between individual driver genes and gene sensitivity scores were identified using an ANOVA model that incorporated both tissue type and mutational status as covariates, similar to the method previously developed for identifying pharmacogenomic interactions in cancer cell line panels (Cokelaer et al., 2018; Iorio et al., 2016). As recent work (Behan et al., 2019) has highlighted that some dependencies (e.g. WRN) can be associated with microsatellite instability rather than individual driver genes, we also incorporated microsatellite instability (Ghandi et al., 2019) as a covariate in our model. The model had the form ‘gene_X_sensitivity ~MSI_status + C(Tissue) + driver_gene_Y_status’ and was used to test the association between each recurrently mutated driver gene Y and all gene sensitivity scores X assayed in a given dataset. Driver genes were included in this analysis if they were functionally altered in at least five cell lines in the dataset being analysed. Correction for multiple hypothesis testing was performed using the Benjamini and Hochberg, 1995 false discovery rate.

### Identifying genetic dependencies common to multiple datasets

When comparing a pair of datasets, we used one dataset as a discovery dataset and a second as a validation set, as outlined in Figure 1C. The discovery analysis was limited to the set of interactions that could be tested in both datasets, that is associations between the set of sensitivity scores for genes screened in both studies and the set of driver genes recurrently altered in both studies. An initial set of genetic interactions was identified in the discovery dataset at a specific FDR threshold and these associations were then tested in the validation set. We considered interactions to be reproduced in the validation dataset if: (1) the FDR was less than the threshold; (2) the uncorrected p-value was <0.05 and; (3) the sign of the association (sensitivity/resistance) was the same in both discovery and validation set. A FDR of 0.2 was used for all analysis presented in the main text but additional FDR thresholds (0.1, 0.3) were tested to ensure that all findings were robust to the exact choice of FDR (Figure 5—figure supplement 1).

我们使用R来进行所有的统计分析。对于每个驱动基因/靶基因组合，我们将携带驱动基因可能发生功能改变的细胞系与该基因没有改变的细胞系进行比较，并测试有功能改变的细胞系是否对抑制该基因的RNAi试剂更敏感。这是用单侧的曼-惠特尼U检验来检验的。在以前的出版物中，已经使用了各种替代的双样本测试，包括中位数互换测试（Brough等人，2011b；Campbell等人，2016）和基于互信息的测量（Cowley等人，2014）。Mann-Whitney U检验对CancerGD来说有很多优势--它计算迅速，而且不假设每个基因的分数是正态分布。后者很重要，因为它意味着该测试可以统一用于使用不同评分方案的多个来源的功能缺失筛选。对于所有的筛选，我们使用作者提供的评分方案（Marcotte等人的zGARP（Marcotte等人，2012；Marcotte等人，2016），Cowley等人的ATARIS表型评分（Cowley等人，2014），Campbell等人的稳健Z评分（Campbell等人，2016），以及Wang等人的CS评分（Wang等人，2017））。如同（Marcotte等人，2012），我们对（Marcotte等人，2012）的zGARP分数进行Z分数归一化，以便合理比较各细胞系的分数。除了Mann-Whitney U检验的P值外，我们还计算每个依赖性的共同语言效应大小（CLES）。CLES相当于ROC曲线下的面积和优越性概率，表示在特定驱动基因上有改变的细胞系比没有改变的细胞系对特定RNAi试剂更敏感的概率。在数据库中，我们存储了所有名义上显著的依赖关系（P<0.05），CLES R为0.65。在少数情况下，一个基因有多个ATARIS分数--当存储CGD时，我们将具有较低P值的ATARIS分数纳入其中。









